python/ray/tests/test_logging.py

```diff
@@ -363,6 +363,23 @@ def component_file_only_one_log_entry(component):
             f"backup count {backup_count}, file count: {file_cnt}"
         )
 
```

Comment:
We should combine `test_log_rotation` and `test_log_rotation_disable_rotation_params` into a single test with a bool rotate parameter.